Components/RoundButton.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { withTheme, TouchableRipple } from 'react-native-paper';
+
+function RoundedButton(props) {
+    const { colors } = props.theme;
```

Comment:
`const { theme: {colors}, text} = props` allows you to just say `{text}` on line 12.